src/index.ts

```diff
@@ -38,7 +40,10 @@ export interface VueLoaderOptions {
 
 let errorEmitted = false
 
-const loader: webpack.loader.Loader = function(source: string) {
+export default function loader(
```

Comment:
This would require `require('vue-loader').default`? Does webpack have built-in esModule default interop?

Reply:
Yeah, webpack uses `loader-runner` to run loaders, and it supports `.default` https://github.com/webpack/loader-runner/blob/master/lib/loadLoader.js#L26

Or we can provide another `.cjs` entry if compatibility is a concern.
But I don't think it's necessary, as `vue-loader` is only supposed to be called by webpack.